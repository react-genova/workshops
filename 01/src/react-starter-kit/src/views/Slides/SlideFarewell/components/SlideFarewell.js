//@flow
import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
    font-family: Arial, Helvetica, sans-serif;
    display: flex;
    justify-content: center;
    flex-direction: column;
    align-items: center;
`;

const Title = styled.div`
    font-size: 7em;
`;

const SubTitle = styled.div`
    font-size: 3em;
    color: #808080;
    padding-top: 0.3em;
`;

const Separator = styled.div`
    margin: 6em;
    width: 75%;
    border: 0.035em solid #C8C8C8; //#264d73
`;

const SlideFarewell = () => (
    <Container>
        <Title>Thank You All</Title>
        <Separator />
        <SubTitle>{'https://www.facebook.com/reactgenova'}</SubTitle>
        <SubTitle>{'reactgenova@gmail.com'}</SubTitle>
        <SubTitle>{'https://github.com/react-genova'}</SubTitle>
    </Container>
);

export default SlideFarewell;