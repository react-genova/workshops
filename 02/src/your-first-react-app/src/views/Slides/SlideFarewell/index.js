//@flow
export {default} from './components/SlideFarewell';