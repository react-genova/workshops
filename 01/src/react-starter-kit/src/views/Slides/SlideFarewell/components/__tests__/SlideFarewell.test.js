//@flow
import React from 'react';
import { shallow } from 'enzyme';
import SlideFarewell from '../SlideFarewell';

describe('Slide Farewell', () => {

    it('snapshot check', () => {
        const component = shallow(<SlideFarewell />);
        expect(component).toMatchSnapshot();
    });

});