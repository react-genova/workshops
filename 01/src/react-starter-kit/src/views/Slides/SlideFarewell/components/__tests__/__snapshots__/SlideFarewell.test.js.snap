// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide Farewell snapshot check 1`] = `
<styled.div>
  <styled.div>
    Thank You All
  </styled.div>
  <styled.div />
  <styled.div>
    https://github.com/react-genova
  </styled.div>
</styled.div>
`;
